src/main/java/com/faforever/moderatorclient/api/FafApiCommunicationService.java

```diff
@@ -12,18 +12,16 @@
 import com.faforever.moderatorclient.mapstruct.CycleAvoidingMappingContext;
 import com.github.jasminb.jsonapi.JSONAPIDocument;
 import com.github.jasminb.jsonapi.ResourceConverter;
+import com.github.jasminb.jsonapi.exceptions.DocumentSerializationException;
 import lombok.Getter;
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
+import org.jetbrains.annotations.Nullable;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.ApplicationEventPublisher;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.HttpHeaders;
-import org.springframework.http.HttpMethod;
-import org.springframework.http.MediaType;
-import org.springframework.http.ResponseEntity;
+import org.springframework.http.*;
```

Comment:
In before Brutus asking to list imports explicitly

Reply:
Intellij adds them back in when I remove them

Reply:
yeah you have to turn it off in intellij https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports